doc/tutorial/index.rst

```diff
@@ -7,7 +7,7 @@ Tutorials
     You can find some examples of full Arbor simulations in the ``python/examples`` directory of the
     `Arbor repository <https://github.com/arbor-sim/arbor>`_.
 
-    The examples use ``pandas`` and ``seaborn`` for analysis and plotting which are expected to be
+    The examples use ``pandas``, ``seaborn`` and ``LFPykit`` for analysis and plotting which are expected to be
```

Comment:
`seaborn` and `pandas` are actually not imported in the script. I suppose the plot is not possible/hard to make with Seaborn primitives?

Reply:
True, I don't use seaborn or pandas in the corresponding example, but the other examples do. One could set some `plt.rcParams` to make the plot more seaborn like I imagine (that is, if your main concern is that plots should have similar style across the documentation).

Reply:
I've updated the plot setting `plt.style.use('seaborn')`.